client/src/components/AuthForm/AuthForm.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useState } from 'react'
+
+import { auth } from '../../stores/auth.js'
+import { TextInput } from '../TextInput/TextInput.js'
+import styles from './AuthForm.module.css'
+
+export const AuthForm = (): JSX.Element => {
+  const [login, setLogin] = useState('admin')
+  const [password, setPassword] = useState('admin')
+
+  const handleLoginChange = useCallback(event => {
+    setLogin(event.target.value)
+  }, [])
+
+  const handlePasswordChange = useCallback(event => {
+    setPassword(event.target.value)
+  }, [])
+
+  const handleSubmit = useCallback(event => {
+    event.preventDefault()
+    auth({
+      name: event.currentTarget.login.value,
+      password: event.currentTarget.password.value
+    })
+  }, [])
+
+  return (
+    <form onSubmit={handleSubmit} className={styles.form}>
+      <div className={styles.field}>
+        <label className={styles.label}>Login:</label>
+        <TextInput
+          id="login"
+          label="login"
+          placeholder="Enter your login"
```

Comment:
`placeholder` according to spec, should be used for example, not for label or description

We do not need `placeholder` here

Reply:
It doesn't seem to be quite so, – according to [this spec](https://html.spec.whatwg.org/multipage/input.html#the-placeholder-attribute), "a hint could be a sample value or a brief description of the expected format".

But, since we have the label here, I agree that we can omit the placeholder here, thanks!